### PR TITLE
Add light theme CSS file

### LINKED ISSUE #2935: Create a new light-theme.css file
For the third step in the theme implementation process, a new light-theme.css file will be created to house the variables and styles specific to the light theme.

### PR DESCRIPTION
**Pull request by mathjazz**
Fix #2935. In addition to adding the light theme, this patch also includes consistency changes on the dark theme.

**Note**: after deployment, a small change is required to the homepage contents stored in the database, because a Mozilla logo in the footer has been moved from HTML to CSS. I've already done it on stage to where this patch is currently deployed.

**How to test themes in Pontoon?**
1. Go to https://mozilla-pontoon-staging.herokuapp.com/settings/ and select Light theme.
2. Explore Pontoon and comment here if you notice anything unusual. See a list of pages to check below.
3. Go to [Settings](https://mozilla-pontoon-staging.herokuapp.com/settings/) again, select Dark theme and repeat the test. We've also made minor changes to the Dark theme.

**What pages to test?**

1. Dashboards:
- https://mozilla-pontoon-staging.herokuapp.com/projects/
- https://mozilla-pontoon-staging.herokuapp.com/projects/firefox/
- https://mozilla-pontoon-staging.herokuapp.com/teams/
- https://mozilla-pontoon-staging.herokuapp.com/fr/
- https://mozilla-pontoon-staging.herokuapp.com/fr/firefox/

2. User pages:
- https://mozilla-pontoon-staging.herokuapp.com/contributors/
- https://mozilla-pontoon-staging.herokuapp.com/profile/
- https://mozilla-pontoon-staging.herokuapp.com/settings/
- https://mozilla-pontoon-staging.herokuapp.com/notifications/

3. Admin:
- https://mozilla-pontoon-staging.herokuapp.com/admin/
- https://mozilla-pontoon-staging.herokuapp.com/admin/projects/firefox/
- https://mozilla-pontoon-staging.herokuapp.com/admin/projects/tutorial/strings/

4. Other assets:
- https://mozilla-pontoon-staging.herokuapp.com/machinery/
- https://mozilla-pontoon-staging.herokuapp.com/terms/
- https://mozilla-pontoon-staging.herokuapp.com/insights/
- https://mozilla-pontoon-staging.herokuapp.com/sync/
- https://mozilla-pontoon-staging.herokuapp.com/ (in [Settings](https://mozilla-pontoon-staging.herokuapp.com/settings/) select "Default Homepage" as your Homepage)
- Add-On Promotion banner

5. Translate View:
* https://mozilla-pontoon-staging.herokuapp.com/fr/firefox/all-resources/

**Known blocking issues:**

- [ ] Theme toggle in the Profile Menu of the Translate View is not yet available
- [ ] [FOUC](https://en.wikipedia.org/wiki/Flash_of_unstyled_content) when Match system is used and the system theme is Light
- [x] In Light theme, Translated status color doesn't have sufficient contrast when used `with font-weight: 300` ([more](https://github.com/mozilla/pontoon/pull/2997#issuecomment-1775273074))
- [x] In Light theme, Pretranslated status color has inappropriate contrast in dashboard progress bar hover
- [x] In Light theme, Missing status color is too dark
- [x] In Light theme, Translation Time Filter colors are too light, because they are hardcoded
- [x] In Light theme, Machinery diff and String list/Concordance search highlights are too dark
- [x] In Dark theme, Priority star in heading needs more contrast
- [x] In Light theme, (manually sent) Notification message color needs more contrast
- [x] Delete button style in the Database project String admin is broken

**Known non-blocking issues:**

- [ ] Interactive Tour colors are hardcoded (light)
- [ ] https://ui-avatars.com/ (shown if no Gravatar image found) colors are hardcoded (dark)
- [ ] Separate main border in two (too dark when between dark and light background on dark theme)
- [ ] Light-theme: Insights chart backgrounds don't work too well with the light theme

**Issues not related to this patch (i.e. issues to be filed):**
- [ ] Buttons in translate view have borders and inputs don't (it's the opposite elsewhere)
- [ ] Welcome to Locale X panel is invisible
- [ ] De-duplicate Notification, Profile menu and Sign in buttons in the header (use React on Dashboards)

### CODE REVIEW

File: pontoon/homepage/templates/homepage_content.html [hunk 1]
```diff
@@ -109,7 +109,7 @@ <h2>Join us.</h2>
       <div class="footer flex">
         <div class="flex-col-3 mozilla">
           <a href="https://mozilla.org/">
-            <img class="logo" src="static/img/moz-logo.svg" />
+            <div class="logo"></div>
```
Comment (mathjazz) [inline pontoon/homepage/templates/homepage_content.html:112]: This needs to be changed in the DB as well.

**Comment by mathjazz** [comment]
Forgot to include changes to the `/translate` module.

**Comment by mathjazz** [comment]
Things to test (just dropping it here for now):

1. Dashboards:
- https://mozilla-pontoon-staging.herokuapp.com/projects/
- https://mozilla-pontoon-staging.herokuapp.com/project/firefox/
- https://mozilla-pontoon-staging.herokuapp.com/teams/
- https://mozilla-pontoon-staging.herokuapp.com/fr/
- https://mozilla-pontoon-staging.herokuapp.com/fr/firefox/

2. User pages:
- https://mozilla-pontoon-staging.herokuapp.com/contributors/
- https://mozilla-pontoon-staging.herokuapp.com/profile/
- https://mozilla-pontoon-staging.herokuapp.com/settings/
- https://mozilla-pontoon-staging.herokuapp.com/notifications/

3. Admin:
- https://mozilla-pontoon-staging.herokuapp.com/admin/
- https://mozilla-pontoon-staging.herokuapp.com/admin/project/firefox/
- https://mozilla-pontoon-staging.herokuapp.com/admin/project/firefox/strings/

4. Other assets:
- https://mozilla-pontoon-staging.herokuapp.com/machinery/
- https://mozilla-pontoon-staging.herokuapp.com/terms/
- https://mozilla-pontoon-staging.herokuapp.com/insights/
- https://mozilla-pontoon-staging.herokuapp.com/ (Homepage)
- Add-On Promotion banner

5. Translate View:
* https://mozilla-pontoon-staging.herokuapp.com/fr/firefox/all-resources/
(includes Welcome to Locale X panel and Interactive Tour)

Code changes are also made to the dark theme, so both should be tested.

**Comment by mathjazz** [comment]
A list of known issues:

- [ ] P1: FOUC system theme selected
- [ ] P5: Interactive Tour colors are hardcoded
- [ ] P5: Translation Time Filter colors are hardcoded
- [ ] P5: https://ui-avatars.com/ (shown if no Gravatar image found) colors are hardcoded

**Comment by flodolo** [comment]
Did you give a try to the accessibility checks for contrast in Firefox devtools?

The MOVE ALL buttons in locale selectors (e.g. at the end of the settings page) seem too light, and hard to read. Same for the CANCEL button. In general, when the text is `--light-green-1` and `font-weight: 300`. The same color is hard to read in the LOCALES or MACHINERY tab (number of matches).

Project header, completion chart, but also completion bar in the list of projects: the "incomplete" part uses a very dark gray like in the dark theme, and it feels strange.

The `PRE` label in the completion bar hover is too light (`--neon-green`).

**Comment by flodolo** [comment]
The [fxa landing page](https://mozilla-pontoon-staging.herokuapp.com/accounts/fxa/login/) used by the add-on is completely borked.

**Comment by flodolo** [comment]
> Thanks for pointing it out - looks like [0bb0752](https://github.com/mozilla/pontoon/commit/0bb0752add1ca4884dd68f6c215ebd139db6675a) wasn't deployed to stage for some reason. I re-deployed this branch, which also includes a few CSS fixes and a theme toggle in the profile menu in all views but the translate view.

The page always uses dark theme for me?

**Comment by mathjazz** [comment]
Correct. Dark theme is default and if you are not logged in we use the default.

**Comment by flodolo** [comment]
> Correct. Dark theme is default and if you are not logged in we use the default.

But I'm already logged in, and my profile is set to use the light theme. If I click projects I get light theme.
I get it's an edge case, but the behavior is incorrect.

**Comment by mathjazz** [comment]
Then this is out of scope of this issue - we should not be showing a Sign In page to logged in users.

**Pull request merged**